lineage/Analyze.py

```diff
@@ -31,7 +31,7 @@ def Analyze_list(Population_list, num_states, **kwargs):
     return tHMMobj_list, pred_states_by_lineage_by_conc, LL
 
 
-def run_Analyze_over(list_of_populations, num_states, parallel=True, **kwargs):
+def run_Analyze_over(list_of_populations, num_states, parallel=True, atonce=False, **kwargs):
```

Comment:
Currently `fit_list` always runs in a combined way so long as two or more conditions are included. Should it see this option?

Reply:
The reason I added this argument was that `run_Analyze_over` was being used to run Analyze, which puts the given population in another square bracket since it is assuming it is not given a list of populations (please see line 10-13). I did this so we call `Analyze_list()` without passing through `Analyze()`. Please let me know if you were referring to another point.

Reply:
Could it just check whether it was given a list, then?

Reply:
done.

Reply:
Doing this made some other problem. I couldn't figure out what is wrong. The error is with line 71 in the Analyze file. It says `lineage object is not subscriptable`. 
![image](https://user-images.githubusercontent.com/34812750/102556701-5caf9580-407e-11eb-94b2-d0310d0db34f.png)
I tried printing the `output` and the `prom_holder`, but I am not sure what `.result()` does.

Reply:
I think this is because Analyze always expects population to be a list. Put population in a list, `[population]` when you directly call it.

Reply:
But if it should be a list, then the line that chooses whether the population goes to `Analyze_list()` or `Analyze()` would be useless.